build/csslint/xfail.list

```diff
@@ -55,6 +55,7 @@ apps/settings/style/settings.css 0 9
 apps/settings/style/settings_large.css 1 1
 apps/settings/style/screen_lock.css 0 2
 apps/settings/style/downloads.css 0 6
+apps/sms/views/conversation/style/message.css 2 0
```

Comment:
@julienw Yes it resides in the parser-lib https://github.com/CSSLint/parser-lib/blob/master/src/css/Properties.js